### PR TITLE
Link Updates

### PR DESCRIPTION
**Pull request by Aeronautical-Eng**

### CODE REVIEW

**Review by pwab — CHANGES_REQUESTED** [review]
As far as I can see you tried to include this map into the repository itself instead of linking to it. IMHO this shouldn't be done because this is a link-list.

Also files like `Units 1.pdf` and `convert.exe` (wtf!?) don't belong here.

The link to the NASA FEMCI book seems to be okay. The rest should be removed.

Also you should try to keep your commit history clean.

File: README.md [hunk 1]
```diff
@@ -108,7 +108,7 @@ I got inspiration from the countless awesome lists in github.
 &emsp; <sub><sup>ISBN: 9780262526548</sup></sub>
 
 #### <a name="MechEng-Map"></a>Mechanical Engineering Knowledge Maps
-* [MECHANICAL ENGINEERING DESIGN](http://www.engineer-assist.com/KnowledgeMap/MechEng_KnowledgeMap_4_files/png_1.htm), knowledge maps with selectable links
+* [MECHANICAL ENGINEERING DESIGN](http://htmlpreview.github.io/?https://github.com/MechEngKnowledgeMap/awesome-mecheng/blob/gh-branches/docs/index.htm), knowledge maps with selectable links
```
Comment (Clockmaker) [inline README.md:111]: the existing link seems fine

**Review by Clockmaker — CHANGES_REQUESTED** [review]
You are still pushing bogus files (pdf, html, txt etc...)

https://github.com/htmlpreview/htmlpreview.github.com doesn't work on my browser.

File: README.md [hunk 2]
```diff
@@ -146,7 +146,14 @@ I got inspiration from the countless awesome lists in github.
 * [thang010146](https://www.youtube.com/user/thang010146/videos), 2100+ animated mechanisms (downloadable) `youtube`
 * [DMG Lib](http://www.dmg-lib.org/dmglib/main/portal.jsp), Digital Mechanism and Gear Library
 * [KMODDL](http://kmoddl.library.cornell.edu/e-books.php), collection of mechanism and machines `Cornell.edu`
-
+* [Engineering ToolBox](https://www.engineeringtoolbox.com/)
+* [Roymechx](http://www.roymech.co.uk/ )
+* [Thermopedia](http://www.thermopedia.com/)
+* [In Tech open](http://www.intechopen.com/)
```
Comment (Clockmaker) [inline README.md:152]: intechopen is already included under *Open Courses*

**Review by Clockmaker — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (Clockmaker) [reply README.md:152]: http://www.roymech.co.uk/ is under development, but the author passed away...
I don't know how to feel about that, but most of the content seems fine.

**Comment by Clockmaker** [comment]
From what I've seen you are attempting to clone http://www.engineer-assist.com/KnowledgeMap/ here.
I'm pretty sure it's against the GPL license, but I can be wrong.

Maybe you can start a different repo and link that instead.
*By the way* all the link on the graph are dead.

**Comment by Aeronautical-Eng** [comment]
Dear Mukunda

Yes some of the links on the http://www.engineer-assist.com/KnowledgeMap/ are
dead, that is why I want to clone the site and fix the links.

If some of the links on http://www.engineer-assist.com
<http://www.engineer-assist.com/KnowledgeMap/> work I will link rather to
their site? What do you think? That is why I created a gh-branch.

Thank you,

Kind Regards,

Taeke

**Comment by Aeronautical-Eng** [comment]
I will see if I can get http://www.engineer-assist.com/KnowledgeMap/  to fix their links.

I will also update the links as you suggest.

**Pull request closed without merging**